lms/models/lti_params.py

```diff
@@ -183,23 +183,10 @@ def _apply_canvas_quirks(lti_params, request):
     # To make upgrades seamless we prefer the LTI1.1 version when available
     #
     # http://www.imsglobal.org/spec/lti/v1p3/migr#lti-1-1-migration-claim
-    (
-        "user_id",
-        [f"{CLAIM_PREFIX}/lti1p1", "user_id"],
-    ),
-    (
-        "resource_link_id",
-        [f"{CLAIM_PREFIX}/lti1p1", "resource_link_id"],
-    ),
+    ("user_id", [f"{CLAIM_PREFIX}/lti1p1", "user_id"]),
+    ("resource_link_id", [f"{CLAIM_PREFIX}/lti1p1", "resource_link_id"]),
     # LMS dependant variables that are not part of the "custom" claim
-    (
-        "org_defined_id",
-        [
-            "https://purl.imsglobal.org/spec/lti/claim/launch_presentation",
```

Comment:
Removed this line, this value is not nested there, it's directly on the "http://www.brightspace.com" namespace.